tests/components/ScriptSetup_ToRefsInject.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div>
+    {{ title }}
+  </div>
+  <p>{{ type }}</p>
+  <div v-if="data.length" id="data">Has Data</div>
+</template>
+
+<script setup>
+import { toRefs, ref, inject } from 'vue'
+
+const props = defineProps({
```

Comment:
in TS, this can be written more elegantly using `defineProps<{ title: string}>()`

Reply:
Sure. Do you think there's a use case for including some plain JS tests, too? Ideally we should running tests against a wide variety of configs - not sure type safety is a big deal in these kind of fixtures.

Reply:
It would be nice to have integration tests as a safety net against various config indeed (something like a mini CLI project in TS and in JS, using the built version of VTU). But I think for our unit tests we should be fine if sticking to TS: it's pretty much the hardest thing to get right from our tooling perspective.